src/pages/fareType.tsx

```diff
@@ -107,7 +109,11 @@ const FareType = ({ operator, errors = [], csrfToken }: FareTypeProps & CustomAp
     );
 };
 
-export const getServerSideProps = (ctx: NextPageContext): {} => {
+export const isFareTypeAttributeWithErrors = (
+    fareTypeAttribute: FareType | FareTypeWithErrors,
+): fareTypeAttribute is FareTypeWithErrors => (fareTypeAttribute as FareTypeWithErrors).errors !== undefined;
```

Comment:
Has this code been moved out into typeChecking.ts now?

Reply:
yup have removed it now